src/main/java/frc/robot/DriveBase.java

```diff
@@ -703,37 +793,66 @@ public static class FollowTrajectory extends CommandBase {
                 new PIDController(0, 0, 0), // Right controller (usually the same values as left controller)
                 // vvv this is supposed to be type BiConsumer<Double, Double>
                 this.drivebase::setDriveVoltage, // Voltage biconsumer
-                true, // Should the path be automatically mirrored depending on alliance color. Optional, defaults to true
+                true, // Sh(ould the path be automatically mirrored depending on alliance color. Optional, defaults to true
                 this.drivebase // Requires this drive subsystem
             );
 
         }
-        
+
+        // public PPRamseteCommand rCommand(PathPlannerTrajectory path)
+        // {
+        //     PPRamseteCommand command = new PPRamseteCommand(
+        //         path, 
+        //         this.drivebase::getDeltaPose,
+        //         new RamseteController(),
+        //         this.drivebase.feedforward,
+        //         this.drivebase.kinematics, // DifferentialDriveKinematics
+        //         this.drivebase::getWheelSpeeds, // DifferentialDriveWheelSpeeds supplier
+        //         new PIDController(0, 0, 0), // Left controller. Tune these values for your robot. Leaving them 0 will only use feedforwards.
+        //         new PIDController(0, 0, 0), // Right controller (usually the same values as left controller)
+        //         // vvv this is supposed to be type BiConsumer<Double, Double>
+        //         this.drivebase::setDriveVoltage, // Voltage biconsumer
+        //         true, // Sh(ould the path be automatically mirrored depending on alliance color. Optional, defaults to true
+        //         this.drivebase // Requires this drive subsystem
+        //     );
+
+        //     return command;
+        // }
+
         FollowTrajectory(DriveBase db, String apath, boolean s)
         {
             super();
             this.path = apath;
             this.stop = s;
             this.drivebase = db;
-            this.pcontroller = null;
+            this.ptrajectory = PathPlanner.loadPath(this.path, new PathConstraints(4, 3));
```

Comment:
There's a lot of code here that looks like it was copied from other parts of the code with only minor differences.  Having similar code repeated in multiple locations makes it harder to maintain.  

I would consider creating a method that contains the repeated code and call the method from each of the locations where the code is repeated  in multiple locations.

Reply:
That is zoke's code you will need to talk to her. I agree though.